bugbug/db.py

```diff
@@ -41,10 +42,17 @@ def is_old_version(path):
     return DATABASES[path]["version"] > prev_version
 
 
-def extract_file(path):
+def extract_file(path, compression_type):
     with open(path, "wb") as output_f:
-        with lzma.open(f"{path}.xz") as input_f:
-            shutil.copyfileobj(input_f, output_f)
+
+        if compression_type == "zst":
+            dctx = zstandard.ZstdDecompressor()
+            with open(f"{path}.zst", "rb") as input_f:
+                dctx.copy_stream(input_f, output_f)
+
+        elif compression_type == "xz":
+            with lzma.open(f"{path}.xz") as input_f:
+                shutil.copyfileobj(input_f, output_f)
 
```

Comment:
Add an else where you do `assert False, f'Unexpected compression type: {compression_type}'`